airbyte-integrations/connectors/source-amplitude/source_amplitude/source.py

```diff
@@ -29,10 +30,24 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         except Exception as error:
             return False, f"Unable to connect to Amplitude API with the provided credentials - {repr(error)}"
 
+    @staticmethod
+    def _validate_start_date(start_date):
+        today = pendulum.now()
```

Comment:
nit: I think calling this variable `now` would be more accurate than `today`

Reply:
updated